src/output-json-alert.c

```diff
@@ -493,6 +583,26 @@ static int AlertJson(ThreadVars *tv, JsonAlertLogThread *aft, const Packet *p)
             }
         }
 
+        if (pa->s->file_flags & FILE_SIG_NEED_FILE) {
```

Comment:
Wonder if this should do this unconditionally. If there are files, add them to the record.

Reply:
Should we make fileinfo output in alert records configurable?

Reply:
I hesitate to say yes as it adds another toggle to the configuration.

Probably bundle it under metadata. If alert metadata is false, then don't include it. If true, then include it?

Reply:
Will use metadata config setting and then include all files, regardless of whether the sig needs them.

Reply:
Sounds good.